build/ansible/pmm-docker/post-build.yml

```diff
@@ -27,25 +27,6 @@
         --server-address=127.0.0.1:8443
         --server-insecure-tls
 
-    - name: Reread supervisord configuration
-      command: supervisorctl reread
-      become: true
-      become_user: pmm
-      become_method: su      
-      register: reread_result
-      changed_when: "'No config updates to processes' not in reread_result.stdout"
-
-    - name: See which service configs changed
-      debug: var=reread_result.stdout_lines
-
-    - name: Stop pmm-managed before deleting the database
-      supervisorctl:
-        name: pmm-managed
-        state: stopped
-      become: true
-      become_user: pmm
-      become_method: su
-
     - name: Remove pmm-managed database
```

Comment:
```suggestion
    - name: Remove PMM-managed  PostreSQL database
```

Reply:
Well, `pmm-managed` is what the binary name is, it doesn't have any other "official" name.

Reply:
Updated.